### PR TITLE
fix(DCS): fix dcs backup resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:

1. add attribute backup_id
2. fix page logic

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  fix dcs backup resource
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/dcs/ TESTARGS='-run TestAccDcsBackup_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dcs/ -v -run TestAccDcsBackup_basic -timeout 360m -parallel 4
=== RUN   TestAccDcsBackup_basic
=== PAUSE TestAccDcsBackup_basic
=== CONT  TestAccDcsBackup_basic
--- PASS: TestAccDcsBackup_basic (168.78s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dcs       168.814s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/dcs/resource_huaweicloud_dcs_backup_test.go [hunk 1]
```diff
@@ -60,20 +51,19 @@ func getDcsBackupResourceFunc(cfg *config.Config, state *terraform.ResourceState
...
 			status := utils.PathSearch("status", backup, "")
 			if status == "deleted" {
-				return nil, fmt.Errorf("error get DCS backup by backup_id (%s)", backupId)
+				return nil, fmt.Errorf("error get DCS backup by backup_id (%s)", state.Primary.ID)
 			}
 			return backup, nil
 		}
 		currentTotal += len(backups)
-		if currentTotal == total {
+		if float64(currentTotal) == total {
```
Comment (deer-hang) [inline huaweicloud/services/acceptance/dcs/resource_huaweicloud_dcs_backup_test.go]: Will there be any problems if we directly compare decimals?

Comment (houpeng80) [reply huaweicloud/services/acceptance/dcs/resource_huaweicloud_dcs_backup_test.go]: done

**Comment by houpeng80** [comment]
make testacc TEST=./huaweicloud/services/acceptance/dcs/ TESTARGS='-run TestAccDcsBackup_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dcs/ -v -run TestAccDcsBackup_basic -timeout 360m -parallel 4
=== RUN   TestAccDcsBackup_basic
=== PAUSE TestAccDcsBackup_basic
=== CONT  TestAccDcsBackup_basic
--- PASS: TestAccDcsBackup_basic (158.02s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dcs       158.098s
huawei@ecs-houpeng:/mnt/d/projects/go_projects/src/github.com/huaweicloud/terraform-provider-huaweicloud$ git commit -a --amend -m "fix(DCS): fix dcs backup resource"

**Pull request merged**